# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/test/mochitest.ini
# Commit: f691cde9edaf
# Full Hash: f691cde9edaf7a11fc554bbda5aaf30893d80170
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-07-21 19:39:17
# Description:
#   Bug 1844741 - Consistently use remaining padding when trimming mp3. r=alwu
#   
#   Creating the file was made in the following way:
#   - Create a VBR file
#   - Modify the padding field so that it spans multiple packets (greater than 1152
# ==============================================================================

diff -r 3c7aa15fc461 -r f691cde9edaf dom/media/test/mochitest.ini
--- a/dom/media/test/mochitest.ini	Fri Jul 21 17:28:03 2023 +0000
+++ b/dom/media/test/mochitest.ini	Fri Jul 21 17:55:32 2023 +0000
@@ -788,6 +788,7 @@
   two-xing-header-no-content-length.mp3^headers^
   single-xing-header-no-content-length.mp3
   single-xing-header-no-content-length.mp3^headers^
+  padding-spanning-multiple-packets.mp3
 
 [test_capture_stream_av_sync.html]
 skip-if =